src/app/iteration/iteration-modal/iteration-modal.component.ts

```diff
@@ -0,0 +1,15 @@
+import { Component, ViewChild } from '@angular/core';
+
+
+@Component({
+  selector: 'fab-planner-iteration-modal',
+  templateUrl: './iteration-modal.component.html',
+  styleUrls: ['./iteration-modal.component.scss']
+})
+export class FabPlannerIterationModalComponent {
+  @ViewChild('createUpdateIterationDialog') createUpdateIterationDialog: any;
+
+  openModal() {
```

Comment:
As it is for create and update iteration modal. I rename it to openCreateUpdateModal().